### PR TITLE
add supports i18n

### PR DESCRIPTION
**Pull request by miya0001**
Added following.

* Japanese translation
* Menu `Change Language`

## Workflow

* Add locale to [_config.yml](https://github.com/miya0001/wp-cli.github.com/blob/3ca0ed1f5f8891b13398b67f7b820e71ad74541f/_config.yml#L30-L34).
* Create a directory for locale.
* Copy index.md into the locale's directory.
* Translate!

Related:
https://github.com/wp-cli/wp-cli/issues/3065

### CODE REVIEW

**Comment by shulard** [comment]
Hello !

This PR seems nice but I don't understand why you introduce an element to change the language in the same time the new language ?

I think it's two different features that must not be in the same PR...

**Comment by miya0001** [comment]
@shulard 

Thanks!
I thought that It will be easier to understand if we have an example. 😄

**Comment by shulard** [comment]
Maybe I'm wrong about that but building a `select` element in JavaScript for the language switch is strange to me. Maybe we need to provide valid links to all the translations ?

It's just about that `select` 😄. It force to introduce jQuery, then an Ajax request to retrieve JSON... It's too much for me.

**Comment by miya0001** [comment]
> Maybe we need to provide valid links to all the translations ?

I thought so too about that. 😄 
Links like flag 🇯🇵 ?

The base idea of this PR is from Ghost documentation.
http://docs.ghost.org/translations/

I felt it is easier to add language, but I am not sure. 😓

**Comment by shulard** [comment]
I think you are right about the translation method with a new folder for each lang. Yeah I though about flag or text (with language name) for the links.

WP-Cli is a Jekyll website and I think it's better to add a new variable in the `_config.yml` file to list language then use that variable to build the language switcher in the header.

Take a look at : https://jekyllrb.com/docs/variables/#site-variables

**Comment by miya0001** [comment]
> WP-Cli is a Jekyll website and I think it's better to add a new variable in the _config.yml file to list language then use that variable to build the language switcher in the header.

It is nice idea, I'll do it. 😄

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Contribute
+
+## How to translate
+
+1. Add your language to `[_config.yml](https://github.com/wp-cli/wp-cli.github.com/blob/master/_config.yml)`.
+2. Create a folder under the name of your appropriate language code (IETF language tag). e.g. ja or pt_BR
+3. Copy `index.md` into your language's directory.
+4. Translate `index.md`.
```
Comment (danielbachhuber) [inline CONTRIBUTING.md]: Can we put this in the `README.md` for now? I don't think our `README.md` is so long that we need to consider separate files right now.

**Pull request merged**